Sources/NordicWiFiProvisioner-SoftAP/ProvisionManager.swift

```diff
@@ -7,58 +7,69 @@
 
 import Foundation
 import NetworkExtension
+import OSLog
+
+private extension URL {
+    static let endpointStr = "https://192.0.2.1/"
+    
+    static let ssid = URL(string: "\(endpointStr)wifi/ssid")!
+    static let prov = URL(string: "\(endpointStr)wifi/prov")!
+    
+    static func led(_ ledNumber: Int) -> URL {
+        let url = URL(string: "\(endpointStr)led/")!
+        if #available(iOS 16, *) {
+            return url.appending(component: "\(ledNumber)")
+        } else {
+            return url.appendingPathComponent("\(ledNumber)")
+        }
+    }
+}
 
 open class ProvisionManager {
-    private let endpoint = "https://192.0.2.1/"
     private let apSSID = "mobileappsrules"
     public init() {
     }
     
     public enum ProvisionError: Error {
-        case httpError(Int)
         case badResponse
     }
     
-    open func connect() async throws {
-        let connectedAlready: Bool
-        // If we're connected, this should return a response. True or false.
-        switch await ledStatus(ledNumber: 1) {
```

Comment:
There's no need to send this request:
1. The device can be connected to the DK, but DK sends an Error in response to the 'LED Status' request. And we decide that we are not connected.
2. `NEHotspotConfigurationManager` returns `NSError` with `code == 13` (Already connected). It's instant and serves precisely to check if we are connected.